odoo/addons/base/data/res_country_data.xml

```diff
@@ -411,6 +411,7 @@
             <field name="currency_id" ref="USD" />
             <field eval="593" name="phone_code" />
             <field name="vat_label">RUC</field>
+            <field name='zip_required'>0</field>
```

Comment:
Do you confirm @vbe-odoo ?

Reply:
Not him but it says so in the spec

Reply:
@jco-odoo I can confirm that the zipcode is not required in EC, and it should not be required because many buyers don't really know their zipcodes in EC.
cc' @pepetreshere @stevTresCloud @ren-odoo @fvz-odoo

Reply:
Yeap, indeed most buyers doesn't know their zip codes. I think by default it should be hidden (if some company wants it they could enable it). I thought that by disabling it in the country will just disable it in the website

![Capture 2024-01-12 at 12 25 22](https://github.com/odoo/odoo/assets/14793916/55d49542-2c3a-483c-902f-d881a9347545)